pallets/pallet-bonded-coins/src/curves/mod.rs

```diff
@@ -107,46 +108,26 @@ fn calculate_accumulated_passive_issuance<Balance: Fixed>(passive_issuance: &[Ba
 
 pub(crate) fn convert_to_fixed<T: Config>(x: u128, denomination: u8) -> Result<CurveParameterTypeOf<T>, ArithmeticError>
 where
-	<CurveParameterTypeOf<T> as Fixed>::Bits: TryFrom<u128>,
+	<CurveParameterTypeOf<T> as Fixed>::Bits: TryFrom<U256>,
```

Comment:
We could in principle make this more generic by not fixing the type of x (an `Into<U256>` trait bound or similar is enough) and by making the type of the big integer configurable in the runtime, so you can match it to the chosen balance and fixed types.

The big integer only needs to be the size of the balance type plus the size of the fractional bits of the fixed, so for a `u64` balance and a `I75F53` fixed a `u128` would be sufficient for lossless conversion.

Reply:
Added a TODO in cffd4bd2ae5a18d02722e18abdd6c0d14dc497f8